Batch File/Batch File.sublime-syntax

```diff
@@ -391,7 +391,9 @@ contexts:
         - match: \n
           scope: comment.line.rem.dosbatch
           pop: true
-        - match: '[(|)]'
+          # '[(|)]' should create a illegal comment statements, but don't
+          # https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-xp/bb490986(v=technet.10)?redirectedfrom=MSDN
```

Comment:
The comment is indented so that it's inside the `\n` match. I think it should have one less indent level.

Also, I'm not sure it's that clear what it's trying to convey, especially once the original match rule is gone.

Reply:
All those lines from 394 to 397 can be removed. An empty match is not what we might want to have.